en/lesson-08.md

```diff
@@ -10,31 +10,51 @@ you load the `array` package, which adds more functionality to LaTeX tables, and
 which is not built into the LaTeX kernel only for historic reasons. So put the
 following in your preamble and we're good to go:
 
+<!-- {% raw %} -->
```

Comment:
I cut down the 'raw' parts to those that are necessary, largely as most of our examples don't need it. I've no problem with it being added back in, but just to note why it's absent at present here.